portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,43 @@
+body {
+  margin: 0;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#logo {
+  width: 150px;
+  position: absolute;
+  top: 20%;
+  user-select: none;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#parentdiv {
+  position: relative;
```

Comment:
The organization and CSS for this seems to only be designed for the single element.  For instance it is 150px x 150px but after you've positioned elements absolutely within it it takes up a lot more space, meaning the box model does not accurately represent its contents on the page.

Instead consider creating one div to contain all 7 of your elements, centering it vertically+horizontally on the page using flexbox, and dynamically positioning all 7 elements inside it.